packages/keyring-eth-trezor/package.json

```diff
@@ -76,9 +75,13 @@
     "sinon": "^19.0.2",
     "ts-jest": "^29.0.5",
     "ts-node": "^10.9.2",
+    "tslib": "^2.6.2",
     "typedoc": "^0.25.13",
     "typescript": "~5.6.3"
   },
+  "peerDependencies": {
+    "tslib": "^2.6.2"
```

Comment:
This should have been a `peerDependencies` since the introduction of `@trezor/connect-web`.